portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java

```diff
@@ -80,6 +108,8 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
       commentEntity.setProperty("comment", text);
       commentEntity.setProperty("email", email);
       commentEntity.setProperty("username", username);
+      commentEntity.setProperty("imageUrl", imageUrl);
+      commentEntity.setProperty("imageLabels", imageLabels);
```

Comment:
What happens if imageLabels is null here?

Reply:
I think I should simply return an empty list if an error occurs as I am not implementing a `retry` mechanism or anything to handle `null`?

Reply:
Yep that sounds good!